.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "jsxSingleQuote": true,
```

Comment:
I just tried this out in VS Code and we should delete this line! `"jsxSingleQuote": true,`